app/models/gogovan_order.rb

```diff
@@ -10,10 +10,12 @@ def self.save_booking(booking_id)
   end
 
   def self.place_order(user, attributes)
+    attributes = set_vehicle_type(attributes) if attributes['offerId']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.